sideboard/lib/__init__.py

```diff
@@ -1,152 +1,24 @@
 from __future__ import unicode_literals
-import os
```

Comment:
Two changes here:

1) An internal refactor which moves all of the random utility stuff into ``sideboard.lib._utils`` which is more consistent with how we're already splitting things up into different private sub-modules.  This is the reason why we see all the deletions here; that code was all moved into that module.

2) We added two new utilities, which are documented below.